### PR TITLE
Adding CodiMD to the projects list

### PR DESCRIPTION
**Pull request by ccoenen**

### CODE REVIEW

File: _data/projects/CodiMD.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: CodiMD
+desc: Realtime collaborative markdown notes
+site: https://github.com/codimd/server
+tags:
```
Comment (ritwik12) [inline _data/projects/CodiMD.yml:4]: @ccoenen Hey, isn't there any languages related to this project? Please add if any.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/CodiMD.yml (hunk 1 above)
Comment (ccoenen) [reply _data/projects/CodiMD.yml:4]: I went over the list of common tags and added all that felt appropriate. Thanks for your suggestion!

Comment (ritwik12) [reply _data/projects/CodiMD.yml:4]: Great!!

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@ccoenen Thanks!!

**Comment by ccoenen** [comment]
You're welcome!

**Pull request merged**